platform/feature-api/api/fetch-spec/src/api.ts

```diff
@@ -28,6 +28,7 @@ export interface Response {
     readonly statusText: string;
     readonly type: ResponseType;
     readonly url: string;
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
I'm no TypeScript export, and I'm not sure this helps, but in another place I recently touched, I saw that we're using `Record<string, unknown>` as a type for a generic JavaScript object.

Reply:
good catch! I used `{ [key: string] : string }` on other cases, but this slipped me. guess works too?!